events/processor/index.js

```diff
@@ -0,0 +1,77 @@
+import AWS from 'aws-sdk';
+import uuid from 'uuid';
+
+import Echo from './types/echo';
+
+export default class Processor {
+  constructor(event) {
+    this.kinesis = new AWS.Kinesis();
+    this.event = event;
+    this.types = {
+      'echo': Echo
```

Comment:
unless the keys in an object have special characters I'd just leave the quotes off the keys as you are just causing yourself a couple extra key strokes :)